solutions/security/configure-elastic-defend/elastic-defend-feature-privileges.md

```diff
@@ -39,8 +39,8 @@ For each of the following sub-feature privileges, select the type of access you
 | --- | --- |
 | **Endpoint List** | Access the [Endpoints](/solutions/security/manage-elastic-defend/endpoints.md) page, which lists all hosts running {{elastic-defend}}, and associated integration details. |
 | **Automatic Troubleshooting** |Access [Automatic Troubleshooting](/solutions/security/manage-elastic-defend/identify-antivirus-software-on-hosts.md) to check if your hosts have third-party AV software installed.<br><br>**Note:** In {{stack}} 9.0.0, this privilege is called **Endpoint Insights**. |
-| **Endpoint Exceptions** | Add and use [endpoint exceptions](/solutions/security/detect-and-alert/add-manage-exceptions.md#endpoint-rule-exceptions).<br><br>**Note:** This privilege is only available in {{serverless-short}}. In {{stack}}, it's included within the **Security** privilege. |
-| **Global Artifact Management** | {applies_to}`stack: ga 9.1` Manage global assignment of endpoint artifacts (e.g., trusted applications, event filters) across all spaces and policies. This privilege controls global assignment rights only; privileges for each artifact type are required for full artifact management. |
+| **Endpoint Exceptions** {applies_to}`stack: ga 9.2` {applies_to}`serverless: ga`| Add and use [endpoint exceptions](/solutions/security/detect-and-alert/add-manage-exceptions.md#endpoint-rule-exceptions).<br><br>**Note:** In {{stack}} 9.0.0 and 9.1.0, this privilege is included within the **Security** privilege. |
```

Comment:
Endpoint exceptions and its inclusion in the Security privilege was introduced in who-knows-when - i have no idea. but i am fairly confident that it was like this in ca. 8.6ish versions, maybe even earlier.

based on this, is it okay to say specifically `In stack 9.0.0 and 9.1.0...` here, or would it be better maybe something like `In stack 9.1.0 and earlier...`?